config/lib/express.js

```diff
@@ -49,6 +49,13 @@ module.exports.initLocalVariables = function (app) {
   app.locals.appSettings.fcmSenderId = config.fcm.senderId;
   app.locals.siteAnnouncement = config.siteAnnouncement || { enabled: false };
 
+  // Predefined to be localized
+  app.locals.appSettings.messages = {
+    msgCanHost: 'Yes, I can host!',
+    msgCannotHost: 'Sorry, I can\'t host',
+    msgWriteBack: 'Write back'
+  };
```

Comment:
Let's move these to Angular controller instead. We'll have a proper i18n later on, no need to prepare for it just yet because it might look very different.

Or actually just having them written static would be okay at this point to avoid complicating it.